packages/react-tests/react-vite/src/App.tsx

```diff
@@ -9,6 +9,14 @@ builder.init(getAPIKey());
 // default to not tracking, and re-enable when appropriate
 builder.canTrack = false;
 
+if (
+  typeof window !== 'undefined' &&
+  !window.location.pathname.includes('can-track-false') &&
+  !window.location.pathname.includes('symbol-tracking')
+) {
+  builder.canTrack = true;
```

Comment:
only issue is im not sure if this will work on tests that include client-side navigations? This code should only run a single time when the JS is first loaded into the browser.

Just something to keep in mind

Reply:
got it, so as canTrack=false tests are only on singular pages we don't face this